UI/MemStickScreen.cpp

```diff
@@ -336,64 +337,62 @@ UI::EventReturn MemStickScreen::SetFolderManually(UI::EventParams &params) {
 	// The old way, from before scoped storage.
 #if PPSSPP_PLATFORM(ANDROID)
 	auto sy = GetI18NCategory("System");
-	System_InputBoxGetString(sy->T("Memory Stick Folder"), g_Config.memStickDirectory.ToString(), [&](bool result, const std::string &value) {
+	System_InputBoxGetString(sy->T("Memory Stick Folder"), g_Config.memStickDirectory.ToString(), [&](const std::string &value, int) {
 		auto sy = GetI18NCategory("System");
 		auto di = GetI18NCategory("Dialog");
 
-		if (result) {
-			std::string newPath = value;
-			size_t pos = newPath.find_last_not_of("/");
-			// Gotta have at least something but a /, and also needs to start with a /.
-			if (newPath.empty() || pos == newPath.npos || newPath[0] != '/') {
-				settingInfo_->Show(sy->T("ChangingMemstickPathInvalid", "That path couldn't be used to save Memory Stick files."), nullptr);
-				return;
-			}
-			if (pos != newPath.size() - 1) {
-				newPath = newPath.substr(0, pos + 1);
-			}
-
-			if (newPath.empty()) {
-				// Reuse below message instead of adding yet another string.
-				System_Toast(sy->T("Path does not exist!"));
-				return;
-			}
-
-			Path pendingMemStickFolder(newPath);
-
-			if (!File::Exists(pendingMemStickFolder)) {
-				// Try to fix the path string, apparently some users got used to leaving out the /.
-				if (newPath[0] != '/') {
-					newPath = "/" + newPath;
-				}
-
-				pendingMemStickFolder = Path(newPath);
-			}
-
-			if (!File::Exists(pendingMemStickFolder) && pendingMemStickFolder.Type() == PathType::NATIVE) {
-				// Still no path? Try to automatically fix the case.
-				std::string oldNewPath = newPath;
-				FixPathCase(Path(""), newPath, FixPathCaseBehavior::FPC_FILE_MUST_EXIST);
-				if (oldNewPath != newPath) {
-					NOTICE_LOG(IO, "Fixed path case: %s -> %s", oldNewPath.c_str(), newPath.c_str());
-					pendingMemStickFolder = Path(newPath);
-				} else {
-					NOTICE_LOG(IO, "Failed to fix case of path %s (result: %s)", newPath.c_str(), oldNewPath.c_str());
-				}
-			}
-
-			if (pendingMemStickFolder == g_Config.memStickDirectory) {
-				// Same directory as before - all good. Nothing to do.
-				TriggerFinish(DialogResult::DR_OK);
-				return;
-			}
-
-			if (!File::Exists(pendingMemStickFolder)) {
-				System_Toast(sy->T("Path does not exist!"));
-				return;
-			}
-
-			screenManager()->push(new ConfirmMemstickMoveScreen(pendingMemStickFolder, false));
-		}
+        std::string newPath = value;
```

Comment:
Ah, I see.  First it's multiple statements on the same line, now always spaces.  I'm sure you'll be correcting code to use `FBO* fbo1,* fbo2` syntax soon as well.

-[Unknown]

Reply:
Uh, not really sure what happened here. I might have accidentally edited in VS Code which I was using for some search/replace stuff, which it's better at.

Not intentional to add spaces!